plugins/node/opentelemetry-winston-transport/src/OpenTelemetryTransport.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Logger, logs } from '@opentelemetry/api-logs';
+import * as Transport from 'winston-transport';
+import { VERSION } from './version';
+import { emitLogRecord } from './utils';
+
+export class OpenTelemetryTransport extends Transport {
+  private _logger: Logger;
+
+  constructor(options?: Transport.TransportStreamOptions) {
+    super(options);
+    this._logger = logs.getLogger('@opentelemetry/winston', VERSION);
```

Comment:
to my understanding the logger returned here has `includeTraceContext` set to `true`.
Does this fit well to the winston instrumentation which adds tracecontext also into the log line itself?
Isn't this double enrichment?

Reply:
Should we receive the Logger created in the instrumentation here then instead? and create one only if none was provided?

Reply:
My point was not that much about having one or two instances of Logger. It's more about adding traceId to the log line itself and additionally adding it into the export. Isn't this doing the same thing twice?

Reply:
Logger will add traceId of current context or use specific one if part of the actual LogRecord, there should not be any problem here that I can see. 
https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/sdk-logs/src/Logger.ts#L31
includeTraceContext was removed last year
https://github.com/open-telemetry/opentelemetry-js/pull/3817

Reply:
ah ok, this was removed...
Seems removal was only partially as I found at least [this](https://github.com/open-telemetry/opentelemetry-js/blob/f86251d40fbf615be87319c8a1f5643afb820076/experimental/packages/api-logs/src/types/LoggerOptions.ts#L33) and [this](https://github.com/open-telemetry/opentelemetry-js/blob/f86251d40fbf615be87319c8a1f5643afb820076/experimental/packages/sdk-logs/src/config.ts#L33).

Reply:
Yeah, I can create a PR later to deprecate the config value and clean it up so we don't forget to take it out once Logs API is stable